Representable/representable
# Re-organized Entry Form Sections

Resolves #104, #59, #73.

**Key Changes:**
- Entry form sections follow the pencil and paper form more closely (#73).
- Adds disclaimer that form is not submitted to commission and instructions (#59).
- Combines map and geocoder in one section to create an easier transition (#104).
- Moves map last so that users can build context before drawing.
- Updates map fields to make them more accessible.
- Added personal information disclaimer.

**How to test:**
1. Go to entry form.
2. Create an entry. Make sure JS works for section traversal ("Next", "Back", "Hide/Edit Section").
3. Fill form. 
4. Read text.
5. Check that map only toggles after you search for a location in the geocoder.

**To Do/Next Steps:**
1. Discuss content with PGP/VNP. Is our disclaimer clear enough?
2. Is our personal information disclaimer good enough?

**Screenshots:**

<img width="951" alt="Screen Shot 2020-05-22 at 01 17 10" src="https://user-images.githubusercontent.com/8714338/82633803-1bc7cf00-9bca-11ea-951a-a1c7f90a6514.png">
<img width="1140" alt="Screen Shot 2020-05-22 at 01 17 03" src="https://user-images.githubusercontent.com/8714338/82633804-1d919280-9bca-11ea-8c14-7cb08c415c3d.png">
<img width="950" alt="Screen Shot 2020-05-22 at 01 18 41" src="https://user-images.githubusercontent.com/8714338/82633824-326e2600-9bca-11ea-88f2-f64724ab9344.png">
